### PR TITLE
thycotic vault permissions-updater

### PR DESCRIPTION
**Pull request by 4killo**
# Description
https://github.com/jenkinsci/thycotic-vault-plugin
@hansboder @4killo 

https://issues.jenkins-ci.org/browse/HOSTING-817

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

#### Merge permission to GitHub repository
- [x] Check this if newly added person also needs to be given merge permission to the GitHub repo.

### CODE REVIEW

**Comment by slide** [comment]
Please fill out the PR template, it is there for a reason. Provide all information requested.

**Comment by 4killo** [comment]
> Please fill out the PR template, it is there for a reason. Provide all information requested.

done

File: permissions/plugin-thycotic-vault.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "thycotic-vault"
+github: "jenkinsci/thycotic-vault-plugin"
+paths:
+  - "com.thycotic/plugins/thycotic-vault"
```
Comment (slide) [inline permissions/plugin-thycotic-vault.yml]: You need to replace all `.` with `/`. Also, it looks like your `groupId` is just `com.thycotic` and your `artifactId` is `thycotic-valut`, so your path would be `com/thycotic/thycotic-vault`

Comment (4killo) [reply permissions/plugin-thycotic-vault.yml]: Fixed!!

**Pull request merged**